e2e/testcases/custom_resources_test.go

```diff
@@ -119,7 +120,8 @@ func TestCRDDeleteBeforeRemoveCustomResourceV1Beta1(t *testing.T) {
 
 	err = nt.ValidateMetrics(nomostest.SyncMetricsToLatestCommit(nt), func() error {
 		// Validate reconciler error metric is emitted.
-		return nt.ReconcilerMetrics.ValidateReconcilerErrors(nomostest.DefaultRootReconcilerName, 1, 1)
+		pod := nt.GetDeploymentPod(nomostest.DefaultRootReconcilerName, configmanagement.ControllerNamespace)
+		return nt.ReconcilerMetrics.ValidateReconcilerErrors(pod.Name, 1, 1)
```

Comment:
Is there a generic case we can add to `nt.ValidateReconcilerErrors` to support `(1, 1)`?